app.py

```diff
@@ -0,0 +1,54 @@
+"""
+This is the main code of my program, Guess the number !!!
+"""
+import random
+import time
+
+
+# Functions
+def get_guess_Player():
+       print("##### Player's turn ####")
+       time.sleep(2)
+       number= int(input("I'ts your turn, please guess the number, write it down here:"))
+       print(number)
+       return number
+
+def get_guess_computer():
+      print("##### Computer's turn ####")
+      time.sleep(2)
+      computerNumber= random.randint(1, 100)
+      print(computerNumber)
+      return computerNumber
+   
+def mainFunction(secretNumber=None):
+    if secretNumber is None:
+        secretNumber= random.randint(1, 100)
+    flag= True
+    while flag:
+        player_guess= get_guess_Player()
+        time.sleep(2)
+        if player_guess == secretNumber:
+            print("Congratulations! You won...")
+            flag= False
+            break
+        else:
+            if player_guess > secretNumber:
+                print('Too high')
+            else:
+                print('Too low')
```

Comment:
You could write a function for this logic.
The function could take the guess and secret number as parameters and compare them, then return a boolean or a tuple with a boolean and a string to say if its high or low or correct.
you can use this function here and below for the computer guess.

Reply:
I'm gona try, last time I had a problem with the break word, because I wasnt in a loop anymore, so that was my problem for separte thos part of my code